packages/ag-charts-community/src/chart/interaction/regionManager.ts

```diff
@@ -307,7 +311,7 @@ export class RegionManager {
             if (newRegion === undefined || !newRegion.properties.canInteraction()) {
                 this.updateFocusIndicatorRect(undefined);
             } else {
-                event.interactionEvent.sourceEvent.preventDefault();
+                event.sourceEvent.consume();
```

Comment:
See other GitHub comment. Instead of changing the behaviour of `consume()`, it would be better to keep this line as-is, but add something similar in `RegionManager.onNav()`:

```diff
     private onNav(event: KeyNavEvent<'blur' | 'nav-hori' | 'nav-vert' | 'submit'>) {
         const focusedRegion = this.getTabRegion(this.currentTabIndex);
+        if (event.type !== 'blur) {
+            event.interactionEvent.sourceEvent.preventDefault();
+        }
         this.dispatch(focusedRegion, event);
    }
```